.github/workflows/holochain-build-and-test.yml

```diff
@@ -41,10 +46,17 @@ jobs:
           - cmd:
               pkgs:
                 - build-holochain-build-crates-standalone
-                - build-release-automation-tests
-                - build-release-automation-tests-repo
                 - build-holochain-tests-static-all
-              extra_arg: "--override-input versions ./versions/0_2 --override-input holochain ${{ inputs.repo_path }}"
+                - build-release-automation-tests-unit
```

Comment:
I've also made changes to this matrix as part of the cachix work, one of us will have to rebase here